src/screens/about/components/comparison.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Link } from "react-router";
+// Common
+import LOGO from "../../../../static/logotype.svg";
+import VLOGO from "victory-docs/static/logotype-hero.svg";
+
+class ComparisonChart extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>When should I use FormidableCharts versus Victory?</h2>
+        <p>We wanted to provide developers with multiple charting options for their projects. FormidableCharts is
+         quick and easy to implement with minimal muss and fuss around styling, theming, and composing components.
+         However, Victory is much more flexible and powerful, and allows users to develop pretty much any kind of
+         chart configuration that they could need. Here's a chart to help you figure out what's best for your
+         project.</p>
+
+        <div className="Grid Grid--guttersSm">
+          <div className="Grid-cell Grid-cell--autoSize">
+            <table className="Table Table--fc">
+              <thead>
+                <tr>
+                  <th>
+                    <div
+                      dangerouslySetInnerHTML={{ __html: LOGO }}
+                      style={{ display: "block", height: "30px" }}
+                    />
+                    <p>
+                      <Link className="btn btn--dark" to="/docs/">
+                        Get Started
+                      </Link>
+                    </p>
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td className="Table-td--same">Native Support</td>
+                </tr>
+                <tr>
+                  <td className="Table-td--same">Animations</td>
+                </tr>
+                <tr>
+                  <td>Pre-Built Charts</td>
+                </tr>
+                <tr>
+                  <td>Series-Style Data</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <div className="Grid-cell Grid-cell--autoSize">
+            <table className="Table Table--v">
+              <thead>
+                <tr>
+                  <th>
+                    <div
+                      dangerouslySetInnerHTML={{ __html: VLOGO }}
+                      style={{ display: "block", height: "30px" }}
+                    />
+                    <p>
+                      <a className="btn btn--gray" href="https://formidable.com/open-source/victory/docs">
+                        Get Started
+                      </a>
+                    </p>
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td className="Table-td--same">Native Support</td>
+                </tr>
+                <tr>
+                  <td className="Table-td--same">Animations</td>
+                </tr>
+                <tr>
+                  <td>Advanced Chart Types</td>
```

Comment:
"Automatic Interactive Events" vs. "Flexible Event System"
"Emphasis on Ease of Use" Vs. "Emphasis on Flexibility"